scripts/thumbnail-images.py

```diff
@@ -3,12 +3,20 @@
 """
 Thumbnail images to a maximum of 320px wide and 160px high
 """
+import argparse
 import glob
 
 from PIL import Image  # pip install pillow
 
 max_size = 320, 160
 
+parser = argparse.ArgumentParser(
+    description="Thumbnail images to a maximum size",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+parser.add_argument("--inspec", default="assets/*.png", help="Input file specification")
```

Comment:
`--inspec` seems like a strange name for the option. What about `--file`?

Reply:
Sure, updated.